### DISCUSSION TITLE
v4.15.1

### DISCUSSION THREAD

**Discussion by dhui** [golang-migrate/migrate#639] [Announcements]
### Database Drivers
* Update gosnowflake from v1.4.3 to v1.6.3 to fix security warnings (#632 & #637). Thanks @nicholas-mainardi and @obitech for reporting the issue!
* Add unofficial support for AWS Keyspaces to the Cassandra DB driver by `DELETE`ing the schema version table instead of using `TRUNCATE` thanks to @florentsolt and @Baumanar (#634)

### Docker Image
* Fixed CLI binary file location (https://github.com/golang-migrate/migrate/issues/625) thanks to @Fontinalis (https://github.com/golang-migrate/migrate/pull/628)

### Builds
* Update dktest to v0.3.7 to fix security warnings (#632). Thanks @nicholas-mainardi for reporting the issue!

<hr /><em>This discussion was created from the release <a href='https://github.com/golang-migrate/migrate/releases/tag/v4.15.1'>v4.15.1</a>.</em>

**Comment by ixry** [comment]
golang.org/x/text@v0.3.2 possibility to trigger an infinite loop in encoding/unicode could lead to ...
go.mod has "github.com/jackc/pgconn v1.8.0" and it relies on golang.org/x/text@v0.3.2
